test-requirements.txt

```diff
@@ -5,3 +5,4 @@ nose
 nosexcover
 openstack.nose_plugin
 nosehtmloutput
+mock>=1.0
```

Comment:
I know. The question is: the bounds used to be
```
py26: mock<1.1,>=1.0
```
So what happens when `mock >= 1.1` exists on a Python2.6 system? Will things fail? Why was the upper bound set for Python2.6 and not for Python2.7?